pkg/plugins/ifaces.go

```diff
@@ -110,6 +110,7 @@ type StaticRouteResolver interface {
 
 type ErrorResolver interface {
```

Comment:
I haven't dug in further, but do you think we need to keep `ErrorResolver` long term when we have `ErrorTracker`?